app/views/shared/_notices.html.erb

```diff
@@ -1,8 +1,10 @@
-<div class="container mx-auto">
-  <% flash.each do |msg_type, message| %>
-    <div class="alert <%= bootstrap_class_for(msg_type) %> alert-dismissible fade show" role="alert">
-      <%= message %>
-      <button type="button" class="btn-close" data-bs-dismiss="alert" aria-label="Close"></button>
-    </div>
-  <% end %>
-</div>
+<% if !flash.empty? %>
+  <div class="container mx-auto">
+    <% flash.each do |msg_type, message| %>
```

Comment:
I'm always trying to remove conditionals, so what do you think if we instead rendered the container inside the loop? 

```ruby
<% flash.each do |msg_type, message| %>
  <div class="container mx-auto">
    <%# ... %>
  <% end %>
<% end %>
```

That would let use remove the `if` and keep it simpler.

Usually we won't have more than one flash, but it is possible, so I guess we'd just want to double check that case.

Reply:
That could work.  on the off chance there is more than one it will definitely add some spacing
<img width="1349" alt="Screen Shot 2021-08-11 at 4 56 56 PM" src="https://user-images.githubusercontent.com/70450072/129102450-45a629ad-7501-4aa4-8696-5876c7505ee0.png">

Reply:
Weird. I wonder where all that padding is coming from. I didn't think `container` or `mx-auto` would be adding any?